launcher/container_runner.go

```diff
@@ -114,25 +117,30 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 	logger.Printf("Operator Override Env Vars : %v\n", envs)
 	logger.Printf("Operator Override Cmd      : %v\n", launchSpec.Cmd)
 
-	imageLabels, err := getImageLabels(ctx, image)
+	imageConfig, err := getImageConfig(ctx, image)
 	if err != nil {
-		logger.Printf("Failed to get image OCI labels %v\n", err)
+		return nil, err
 	}
 
-	logger.Printf("Image Labels               : %v\n", imageLabels)
-	launchPolicy, err := spec.GetLaunchPolicy(imageLabels)
+	logger.Printf("Exposed Ports:             : %v\n", imageConfig.ExposedPorts)
+	if err := openPorts(imageConfig.ExposedPorts); err != nil {
```

Comment:
should we defer a cleanup of the iptables rule?

Reply:
Hmmm, I think it's fine if we don't, the table won't persistent anyway.